docs/programming-guide.md

```diff
@@ -1345,14 +1345,17 @@ therefore be efficiently supported in parallel. They can be used to implement co
 MapReduce) or sums. Spark natively supports accumulators of numeric types, and programmers
 can add support for new types.
 
-If accumulators are created with a name, they will be
-displayed in Spark's UI. This can be useful for understanding the progress of
-running stages (NOTE: this is not yet supported in Python).
+As a user, you can create `Accumulators` that are both named and unnamed. Named accumulators will display in Spark's UI along with their running totals during execution. As seen in the image below, a named accumulator (in this instance `counter`) will display
```

Comment:
made these clarifications. I think it is important however to call out that they can be named or unnamed, so I just rephrased that.  Just because something can have a name is not clear enough to me that it can also be unnamed.